chihacknight/state-by-state-decarbonization
# Rename PlaceDetail to StateDetails and Simplify URLs

## Overview

Renames `PlaceDetail` component and template to `StateDetails` to make things clearer now that we know we are making state pages but not counties or cities. Also simplifies URL so instead of `/place/:stateName` it's now just `/:stateName` and then cleans up the two files for `StateDetails` into one.

This is my start to to #20, but I wanted to break it out into a separate PR to prevent merge conflicts and keep things tidy.

### Demo

**Screenshot with new short URL:**

![Screenshot from 2022-03-08 21-47-34](https://user-images.githubusercontent.com/3187531/157369075-d2a83c8c-2917-4950-88ff-80b51300b1c6.png)

## Testing Instructions

- Build the app
- Confirm you can still click the map and see a state view, but at the new URL
